src/resolvers/userPreparedNotificationsList/schema.graphql

```diff
@@ -1,8 +1,10 @@
 type UserPreparedNotification {
   id: ID
   createdAt: DateTime
-  title: String
-  message: String
+  type: String!
```

Comment:
I made it non-null string because I think we should make `key` (type) in templates mandatory